src/main/kotlin/com/github/mitchellolsthoorn/testgenie/evosuite/ResultWatcher.kt

```diff
@@ -10,8 +10,8 @@ import java.io.File
 import java.io.FileReader
 
 
-class EvoSuiteResultWatcher(private val project: Project, private val resultPath: String) : Runnable {
```

Comment:
What was the cause behind changing the class here?

Reply:
I think Lyuben said that he doesn't really like the names that start with Evo since it can be inferred from the package that it is a class related to EvoSuite